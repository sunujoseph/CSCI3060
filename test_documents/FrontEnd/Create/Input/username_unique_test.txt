login
admin1
create
userBS
