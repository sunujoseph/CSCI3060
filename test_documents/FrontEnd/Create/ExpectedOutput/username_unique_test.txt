Enter Username: 
Enter Username For New User: 
Error: User With That Name Already Exists
Enter Username For New User: 